idea.install

```diff
@@ -23,4 +24,12 @@ function idea_install() {
 
   // Install 'openideal_content' module.
   \Drupal::service('module_installer')->install(['openideal_content']);
+
+  // Install pluggins need to ckeditor_media_embed module.
+  $process = new Process('../vendor/bin/drupal ckeditor_media_embed:install');
+  $process->run();
+
+  if (!$process->isSuccessful()) {
+    \Drupal::logger('system')->error(t('An error occured during CKEditor plugins installation.'));
```

Comment:
1) https://www.grammarly.com/blog/occurred-occured-ocurred
2) Let's explain the possible reasons and next steps if this happened. Perhaps a reference to the ckeditor_media_embed instructions will be useful.